### ISSUE TITLE
Unable to register.

### ISSUE THREAD

**Issue by Visham32** [jaylenw/AngularJsTodoApp#38]
When correctly inserted correct credentials, i am unable to gain access to the note area.

![error while registering with correct information](https://cloud.githubusercontent.com/assets/22685311/22701196/74820652-ed76-11e6-8d1c-3742ef09abd8.png)

**Comment by jaylenw** [comment]
Okay i may have introduced a bug when i fixed the previous bug here #30 .

I will check it out. Thanks

**Comment by jaylenw** [comment]
I was able to register a new account, and make notes. Logged out, and with the same account I registered with I was able to log in again. I am not sure how this problem is occurring for you. But then again it sounds like you already created an account with that email and password correct? If so, then click the login button and enter your email and password there. If it works, then that means the user interface / message is not clear enough for users to login and register without confusion. If it doesn't work, I need to do more testing to see why this is broken.
